### PR TITLE
Using Same button for Next and Done

### PR DESCRIPTION
**Pull request by Vishalk91-4**
Fixed and made a single button for both next and done in screen

@llaske could you please review it

### CODE REVIEW

File: v2/js/screens/settings-mysecurity.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const MySecurity = {
 							<div class="security-message" >{{SugarL10n ? SugarL10n.get('SecurityMessage') : ''}}</div>
 							<password-box class="security-password" ref="password" @passwordSet="login"/>
 							<icon-button 
-								id="next-btn"
+								id="next-done-btn"
```
Comment (llaske) [inline v2/js/screens/settings-mysecurity.js:20]: Not sure to understand why it need to be changed

File: v2/js/screens/settings-mysecurity.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ const MySecurity = {
 							<div class="security-message" >{{SugarL10n ? SugarL10n.get('SecurityMessageNew', {"min": 4}) : ''}}</div>
 							<password-box class="security-password" ref="newpassword" @passwordSet="updatePassword"/>
 							<icon-button 
-								id="next-btn"
+								id="next-done-btn"
```
Comment (llaske) [inline v2/js/screens/settings-mysecurity.js:38]: Not sure to understand why it need to be changed

**Comment by Vishalk91-4** [comment]
@llaske 
I changed the id of next-btn and done-btn while merging them, I added it as next-done-btn so next-btn isn't present anywhere
So, I changed it to next-done-btn in those 2 files also

**Comment by llaske** [comment]
Nice thanks

**Pull request merged**